packages/dapp/src/tokenSupport/tokenSchema.json

```diff
@@ -3,36 +3,42 @@
     {
       "tokenContract": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
       "decimals": "18",
-      "symbol": "WETH"
+      "symbol": "WETH",
+      "image": "https://github.com/raid-guild/smart-invoice/blob/tokenImages/weth.png?raw=true"
```

Comment:
is github a good way to host these images? Also we should make sure to protect this branch since it is not necessarily clear from the outside what this branch is up to.

Reply:
i could go the ipfs route if that's preferable; otherwise this seems like a clean/easy way to keep everything in one place with built-in permanence versus an outside hosting option. Open to whatever works best 

in the meantime I'll add a ReadME to the tokenImages branch and rename with something more descriptive for clarity

Reply:
Yeah, I don't know the right answer, just worried about spreading out dependencies in ways that might be fragile. Also, I haven't seen this pattern and wondering if it is common and stable.

Reply:
Although I prefer using a decentralized solution like IPFS or Storj, the one current caveat with using them is that fetching the images from them ( even if we use a dedicated gateway) will be slow. So I am okay with using a centralized service as a current alternative if it allows faster image fetching. But I do agree, this seems a little odd using a separate branch as a storage option. We can possibly consider S3 from AWS and keep the links to those images public. I believe that could be less fragile than making sure no one deletes the wrong branch.

Reply:
Ok, sounds good; maybe firebase then as we're already using it